pkg/agent/ipassigner/responder/arp_responder.go

```diff
@@ -129,3 +168,11 @@ func (r *arpResponder) addIP(ip net.IP) bool {
 	}
 	return !exist
 }
+
+func (r *arpResponder) onLinkUpdate(linkName string) {
+	klog.V(4).InfoS("Received link update event", "name", linkName)
+	select {
+	case r.linkEventCh <- struct{}{}:
```

Comment:
add comment here:
// if an event is already present in the channel, we can drop this new one as we only monitor one link

Reply:
added.